rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/value/ExperienceValue.java

```diff
@@ -1,10 +1,9 @@
 package pl.oaza.warszawa.dor.rekolekcje.api.participants.value;
 
-import lombok.Builder;
-import lombok.Value;
-
 import java.time.ZonedDateTime;
 import java.util.Set;
+import lombok.Builder;
+import lombok.Value;
```

Comment:
\[Checkstyle\] WARNING: Import statement for 'lombok\.Value' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.